### DISCUSSION TITLE
build#configuring-commonjs-dependencies

### DISCUSSION THREAD

**Discussion by GaurangDhorda** [NG-ZORRO/ng-zorro-antd#6027] [Q&A] [answered]
I am including ng-zorro-antd inside angular version 10.2.0. I am following documentation of zorro, and use "ng add ng-zorro-antd", command, after installing successfully, When I am serving application locally using "ng serve -o". I am getting this warning message, 

WARNING in F:\angular\mart\smartshop\node_modules\@ant-design\icons-angular\__ivy_ngcc__\fesm2015\ant-design-icons-angular.js depends on '@ant-design/colors'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Can anyone here, explains what is wrong with this ? And how can i solve this warning message?

**Comment by vthinkxie** [comment, marked as answer]
@ant-design/colors is a commonjs package, but it won't bring any harm.
we will upgrade to support esm later
https://angular.io/guide/build#configuring-commonjs-dependencies

**Comment by GaurangDhorda** [comment]
Thank for letting me know. Nz-Zorro is one of my favorite UI Library.

**Comment by wzhudev** [comment]
We have release @ant-design/icons-angular@10.1.0 which uses @ant-design/colors@4.0.0 that supports ESM. This warning message would be gone the next time you install.
